pkg/commenting/comment.go

```diff
@@ -0,0 +1,5 @@
+package commenting
+
+type Comment struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type Comment should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5025522)